tracer/test/test-applications/azure-functions/Samples.AzureFunctions.V4Isolated.AspNetCore/Samples.AzureFunctions.V4Isolated.AspNetCore.csproj

```diff
@@ -0,0 +1,34 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>net8.0;net9.0</TargetFrameworks>
```

Comment:
Is this correct, we can't/shouldn't run SdkV2 on .NET 6 or .NET 7?

Reply:
Nope, just didn't add them, I'll toss them on it

Reply:
Done, added 6 and 7